### ISSUE TITLE
core/packages not added to archive

### ISSUE THREAD

**Issue by lektriq** [jdaehne/BackupMODX#48]
This issue appears here on:

BackupMODX 3.0.1-pl

Litespeed 5.3.8

MODX 2.6.1 and 2.7.1
PHP 5.6.40 and 7.2.22

When making backup core/packages (permissions, 755) is not being added to the archive.

I note this across several MODX 2.7.1 installations and a 2.6.1 installation.

**Comment by Jako** [comment]
Yes, thats excluded, because the packages folder could get quite large. Why do you want to include that folder?

**Comment by lektriq** [comment]
RecentIy, I had errors thrown at me following a recent backup/transfer and realised lack of core/packages folder was the culprit.

It would be nice to be able to choose to create a full or partial backup. Is this behaviour a recent addition? I didn’t notice this until recently.
